cs3/storage/provider/v1beta1/provider_api.proto

```diff
@@ -665,6 +689,10 @@ message RemoveGrantRequest {
   // REQUIRED.
   // The grant to remove.
   Grant grant = 3;
+  // OPTIONAL.
+  // A lock_id: should the reference be locked and the lock be exclusive,
+  // the stored lock_id MUST be equal to the given value.
+  string lock_id = 4;
```

Comment:
I'm not entirely sure if move/copy need a source as well as a destination lock. I'd merge like this and add another lock_id  in a subsequent PR if necessary.